.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+# Release any branch to Maven Central.
+# This workflow currently assumes that the target branch is ready to be release (i.e. version is correct)
+
+name: Release
+
+on: [ workflow_dispatch ]
```

Comment:
What about running it once a release is created in GH? So that we don't have to create the release + trigger the job? and triggering the release also means we are more likely to target a branch/tag that is actually a release

Reply:
Or maybe we can keep both, upon release and manually

Reply:
Yep let me try this way!

Reply:
Now done https://github.com/finos/symphony-bdk-java/blob/main/.github/workflows/release.yml#L8